PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/com/microsoft/intellij/runner/functionapp/config/FunctionAppRunState.kt

```diff
@@ -82,6 +82,30 @@ class FunctionAppRunState(project: Project, private val myModel: FunctionAppSett
                 AzureRiderSettings.VALUE_COLLECT_ARTIFACTS_TIMEOUT_MINUTES_DEFAULT) * 60000L
 
         val app = getOrCreateFunctionAppFromConfiguration(myModel.functionAppModel, processHandler)
+
+        if (myModel.functionAppModel.isCreatingNewApp && app is FunctionApp) {
```

Comment:
This section updates the runtime settings in Azure (only when creating it from Rider, as we do with web apps).